packages/communication-react/breaking-change-check/checkBreakingChanges.ts

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/* eslint-disable @typescript-eslint/ban-types */
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Why is this linter warning turned off? unused-vars seems like an easy thing to fix up

Reply:
We don't use those variable any where, those assignment just for a purely check of compatibility of "current" and "base"